### ISSUE TITLE
Ecto 3.2 Warnings

### ISSUE THREAD

**Issue by philipgiuliani** [jshmrtn/crontab#62]
After updating to Ecto 3.2, I got the following warnings:
```
warning: function embed_as/1 required by behaviour Ecto.Type is not implemented (in module Crontab.CronExpression.Ecto.Type)
  lib/crontab/cron_expression/ecto_type.ex:2: Crontab.CronExpression.Ecto.Type (module)

warning: function equal?/2 required by behaviour Ecto.Type is not implemented (in module Crontab.CronExpression.Ecto.Type)
  lib/crontab/cron_expression/ecto_type.ex:2: Crontab.CronExpression.Ecto.Type (module)
```

See the last 2 lines of https://github.com/elixir-ecto/ecto/blob/master/CHANGELOG.md#enhancements

**Comment by maennchen** [comment]
Thanks for the report, would you be willing to send a PR?

**Comment by maennchen** [comment]
Released as `1.1.8`
